__test__/components/kytos/inputs/Select.test.js

```diff
@@ -0,0 +1,172 @@
+                {value: "testVal2", description: "testDesc2"},
+                {value: "testVal3", description: "testDesc3"}];
+            wrapper = mount(Select, {
+                props: {
+                    options: testOptions
+                }
+            });
+
+            expect(wrapper.exists()).toBe(true);
+            const [option1, option2, option3] = wrapper.findAll('option');
+            const [optionText1, optionText2, optionText3] = [option1.text(), option2.text(), option3.text()];
+            const [optionValue1, optionValue2, optionValue3] = [option1.element.value, option2.element.value, option3.element.value];
+
+            expect(option1.exists()).toBe(true);
+            expect(option2.exists()).toBe(true)
+            expect(option3.exists()).toBe(true)
+
+            expect(optionText1).toEqual(testOptions[0].description);
+            expect(optionText2).toEqual(testOptions[1].description);
+            expect(optionText3).toEqual(testOptions[2].description);
+
+            expect(optionValue1).toEqual(testOptions[0].value);
+            expect(optionValue2).toEqual(testOptions[1].value);
+            expect(optionValue3).toEqual(testOptions[2].value);
+        });
+
+        test("Select Action", async () => {
+            const testOptions = [{value: "testVal1", description: "testDesc1"},
+                {value: "testVal2", description: "testDesc2"},
+                {value: "testVal3", description: "testDesc3"}];
+            const fn = vi.fn();
+            wrapper = mount(Select, {
+                props: {
+                    options: testOptions
+                }
+            });
+            expect(wrapper.exists()).toBe(true);
+            const mainSelect = wrapper.get('[data-test="main-select"]');
+
+            expect(wrapper.props().hasOwnProperty('action')).toBe(true);
+
+            await wrapper.setProps({ action: fn });
+
+            expect(wrapper.props('action')).toBe(fn);
+
+            await mainSelect.setValue(['testVal1', 'testVal3']);
+
+            expect(fn).toHaveBeenCalledTimes(2);
+            expect(fn).toHaveBeenCalledWith([testOptions[0].value, testOptions[2].value]);
+        });
+    });
+
+    describe("User Interactions", () => {
+        test("Select Data/Use Select", async () => {
+            const testOptions = [{value: "testVal1", description: "testDesc1"},
+                {value: "testVal2", description: "testDesc2"},
+                {value: "testVal3", description: "testDesc3"}];
+            wrapper = mount(Select, {
+                props: {
+                    options: testOptions
+                }
+            });
+            expect(wrapper.exists()).toBe(true);
+            const mainSelect = wrapper.get('[data-test="main-select"]');
+
+            await mainSelect.setValue(['testVal1', 'testVal3']);
+
+            expect(wrapper.vm.selected).toEqual(['testVal1', 'testVal3']);
+        });
```

Comment:
I would add an "empty array" test, and a  select/deselect test.
All the other tests seems OK. Good job!